tests/core_tests/wallet_tools.h

```diff
@@ -68,12 +68,14 @@ class WalletApiAccessorTest
 class wallet_tools
 {
 public:
-  static void gen_tx_src(size_t mixin, uint64_t cur_height, const tools::wallet2::transfer_details & td, cryptonote::tx_source_entry & src, block_tracker &bt);
+  static void gen_tx_src(size_t mixin, uint64_t cur_height, const tools::wallet2::transfer_details & td, cryptonote::tx_source_entry & src, block_tracker &bt, const boost::optional<fnc_accept_output_t>& fnc_accept = boost::none);
   static void gen_block_data(block_tracker &bt, const cryptonote::block *bl, const map_hash2tx_t & mtx, cryptonote::block_complete_entry &bche, tools::wallet2::parsed_block &parsed_block, uint64_t &height);
   static void compute_subaddresses(std::unordered_map<crypto::public_key, cryptonote::subaddress_index> &subaddresses, cryptonote::account_base & creds, size_t account, size_t minors);
   static void process_transactions(tools::wallet2 * wallet, const std::vector<test_event_entry>& events, const cryptonote::block& blk_head, block_tracker &bt, const boost::optional<crypto::hash>& blk_tail=boost::none);
+  static void process_transactions(std::vector<tools::wallet2*> wallets, const std::vector<test_event_entry>& events, const cryptonote::block& blk_head, block_tracker &bt, const boost::optional<crypto::hash>& blk_tail=boost::none);
```

Comment:
`wallets` should be const reference: `const std::vector<tools::wallet2*>& wallets`

Reply:
I've got this warning:
> Clang-Tidy: Parameter 'wallets' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions

Reply:
You probably didn't change it in the function definition. It should be the same in both places.

Reply:
maybe stale IDE analysis. added now